arbitrator/arbutil/src/types.rs

```diff
@@ -13,6 +13,14 @@ use std::{
 #[repr(C)]
 pub struct Bytes32(pub [u8; 32]);
 
+
+impl Bytes32 {
+    pub const fn new(bytes: [u8; 32]) -> Self {
```

Comment:
I don't think you need that.. see comment where you use it

Reply:
Done